doc(release): Release notes for web 23.04.22 (#3585)
centreon/centreon-documentation: versioned_docs/version-23.04/releases/centreon-os.mdx

## Centreon Web

### 23.04.22

Release date: `September 24, 2024`

<details>
  <summary>Bug fixes</summary>

- [API] Fixed access control for monitoring server configuration endpoints.

> - [Authorization] Fixed an issue on ACLs reloading.

Reviewer: ```suggestion
- [Authorization] Fixed an issue on ACL reloading.
```

> - [API] Fixed access control for monitoring servers configuration endpoints.
> - [Authorization] Fixed an issue on ACLs reloading.
> - [Authentication] Fixed incorrect URL redirection issue with OpenId Connect.
> - [Authentication] Rescheduled the task of deleting expired tokens.
> - [Packaging] Fixed invalid permissions on oudated token cron

Reviewer: ```suggestion
- [API] Fixed access control for monitoring server configuration endpoints.
- [Authorization] Fixed an issue on ACLs reloading.
- [Authentication] Fixed incorrect URL redirection issue with OpenID Connect.
- [Authentication] The task that deletes expired tokens is now scheduled more frequently.
- [Packaging] Fixed invalid permissions on the cron file that deletes outdated tokens.
```
